helpme/models.py

```diff
@@ -119,19 +119,19 @@ class Meta:
     def __str__(self):
         return "{0} - {1}".format(self.user.username, self.subject)
 
-    def log_history_event(self, event, user=None, notes=None, isotime=None):
+    def log_history_event(self, action, user=None, notes=None, time=None):
         '''
         "history": [
             {
                 "time": "2020-04-08T23:45:02.225609+00:00",
-                "event": "created"
+                "action": "created"
             }
         ],
         '''
-        if not isotime:
```

Comment:
This should be in iso format and not to be tied to any timezone.  We should look to change the timestamp with a filter in the template to make it readable in a localized format.